docs/website/tds/reference/WCS.html

```diff
@@ -23,7 +23,7 @@ <h2>Which files can be served through the WCS server? </h2>
   <li>Data files must contain gridded data.</li>
   <li>The
     NetCDF-Java Common Data Model must be able to identify the coordinates
-    system used. Check this by opening in the Grid Panel of the <a href="https://www.unidata.ucar.edu/software/netcdf-java/v4.6/webstart/netCDFtools.jnlp">ToolsUI</a>
+    system used. Check this by opening in the Grid Panel of the <a href="https://www.unidata.ucar.edu/netcdf-java/4.6/userguide/webstart/netCDFtools.jnlp">ToolsUI</a>
```

Comment:
Let's just reference the URL on the docs server directly and not rely on the redirect we will put into place.

Reply:
webstart links updated to point to downloads.unidata.ucar.edu.